src/App.tsx

```diff
@@ -4,11 +4,13 @@ import { blue } from '@material-ui/core/colors';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import React from 'react';
 
-import { withNavbar } from './shared/Navbar';
+// import { withNavbar } from './shared/Navbar';
```

Comment:
Get rid of this, seeing as it's still being imported anyways

Reply:
Wat